### PR TITLE
completed task 9

### PR DESCRIPTION
**Pull request by zaw-tun**
Completed route DELETE /api/comments/:comment_id

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -254,3 +254,22 @@ describe("/PATCH /api/articles/:article_id", () => {
...
+      });
+  });
+  it("status:404, responds with no content", () => {
+    return request(app)
+      .delete("/api/comments/100")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.message).toBe("No such comment_id exist yet");
+      });
+  });
```
Comment (Slyose) [inline __tests__/api.test.js:274]: Also worth an invalid id test, e.g. delete("/api/comments/banana")

File: models/articles.model.js [hunk 2]
```diff
@@ -92,10 +92,33 @@ function incVotesById(article_id, inc_votes) {
...
+function deleteComment(comment_id) {
+  return db
+    .query("SELECT * FROM comments WHERE comment_id = $1;", [comment_id])
+    .then((data) => {
+      if (data.rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          message: "No such comment_id exist yet",
+        });
+      }
```
Comment (Slyose) [inline models/articles.model.js:104]: We can also try do the deletion, and if the delete query returns 0 rows assume that there was no comment with that id. That optional improvement will allow us to do this in one query

File: models/articles.model.js [hunk 3]
```diff
@@ -92,10 +92,33 @@ function incVotesById(article_id, inc_votes) {
...
 }
 
+function deleteComment(comment_id) {
+  return db
+    .query("SELECT * FROM comments WHERE comment_id = $1;", [comment_id])
+    .then((data) => {
+      if (data.rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          message: "No such comment_id exist yet",
```
Comment (Slyose) [inline models/articles.model.js:102]: I'd rewrite this error message a bit for clarity - saying something a bit more like 'No comment with specified Id' to make it clear what exactly has gone wrong

File: models/articles.model.js (hunk 2 above)
Comment (zaw-tun) [reply models/articles.model.js:104]: Got it! I could have explored what would be returned if there was nothing to delete.

File: models/articles.model.js (hunk 3 above)
Comment (zaw-tun) [reply models/articles.model.js:102]: Amended.

File: __tests__/api.test.js (hunk 1 above)
Comment (zaw-tun) [reply __tests__/api.test.js:274]: Done. New test will be included in my next task submission. Thank you!

**Pull request merged**